aggsender/flow_pp_test.go

```diff
@@ -241,15 +240,14 @@ func TestGetImportedBridgeExits(t *testing.T) {
 
 	mockProof := generateTestProof(t)
 
-	mockL1InfoTreeSyncer := mocks.NewL1InfoTreeSyncer(t)
-	mockL1InfoTreeSyncer.On("GetInfoByGlobalExitRoot", mock.Anything).Return(&l1infotreesync.L1InfoTreeLeaf{
-		L1InfoTreeIndex:   1,
-		Timestamp:         123456789,
-		PreviousBlockHash: common.HexToHash("0xabc"),
-		GlobalExitRoot:    common.HexToHash("0x7891"),
-	}, nil)
-	mockL1InfoTreeSyncer.On("GetL1InfoTreeMerkleProofFromIndexToRoot", mock.Anything,
-		mock.Anything, mock.Anything).Return(mockProof, nil)
+	mockL1InfoTreeQuery := mocks.NewL1InfoTreeDataQuery(t)
+	mockL1InfoTreeQuery.On("GetProofForGER", mock.Anything, mock.Anything, mock.Anything).Return(
```

Comment:
Optional, consider using the expecter, as it is less error prone:


```suggestion
	mockL1InfoTreeQuery.EXPECT().GetProofForGER(mock.Anything, mock.Anything, mock.Anything).Return(
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/336/commits/b45bfd13336bb63c248f301b3e04bde29ebda5ab